packages/gatsby-plugin-typescript/package.json

```diff
@@ -12,7 +12,7 @@
   "dependencies": {
     "@babel/preset-typescript": "7.0.0-beta.47",
     "@babel/runtime": "7.0.0-beta.47",
-    "babel-plugin-remove-graphql-queries": "^2.0.1-12"
+    "babel-plugin-remove-graphql-queries": "2.0.2-alpha.0"
```

Comment:
not a major version bump?

Reply:
this seems to be v2 specific

Reply:
@calcsam I updated the description to describe the rules behind the version bumps in this PR